lifecycle-image/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Stage 1
```

Comment:
Wonder if it's worth scoping this file to the pipeline by putting it under `.github/workflows/` similar to [pack](https://github.com/buildpacks/pack/tree/master/.github/workflows/delivery).